src/Rector/v11/v2/MigrateFrameModuleToSvgTreeRector.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Ssch\TYPO3Rector\Rector\v11\v2;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Scalar\String_;
+use PHPStan\Type\ObjectType;
+use Rector\Core\Rector\AbstractRector;
+use Ssch\TYPO3Rector\Helper\FilesFinder;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @changelog https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/11.2/Deprecation-93944-FileTreeAsIframeMigratedToSVG-basedTree.html
+ * @see \Ssch\TYPO3Rector\Tests\Rector\v11\v2\MigrateFrameModuleToSvgTreeRector\MigrateFrameModuleToSvgTreeRectorTest
+ */
+final class MigrateFrameModuleToSvgTreeRector extends AbstractRector
+{
+    public function __construct(
+        private FilesFinder $filesFinder
+    ) {
+    }
+
+    /**
+     * @return array<class-string<Node>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [StaticCall::class];
+    }
+
+    public function refactor(Node $node): ?Node
+    {
+        $fileInfo = $this->file->getSmartFileInfo();
+
+        if (! $this->filesFinder->isExtTables($fileInfo)) {
+            return null;
+        }
+
+        if (! $this->nodeTypeResolver->isMethodStaticCallOrClassMethodObjectType(
```

Comment:
Please add a dedicated shouldSkip method for better readability with some early returns